bower.json

```diff
@@ -11,7 +11,7 @@
     "build/react-motion.map"
   ],
   "dependencies": {
-    "react": "^0.14.9 || ^15.3.0"
+    "react": "^15.5.0"
```

Comment:
Since it is only related to tests, maybe worth keeping peer deps as is, but only change devDeps?